pkg/util/helm/bundle.go

```diff
@@ -0,0 +1,75 @@
+package helm
+
+import (
+	"fmt"
+
+	kudo "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/bundle"
+
+	"github.com/helm/helm/pkg/chartutil"
+)
+
+// ToBundle converts the helm chart on disk into a Kudo Bundle
+func ToBundle(folder string) (bundle.Framework, error) {
+	b := bundle.Framework{}
+
+	meta, err := chartutil.LoadChartfile(folder + "/Chart.yaml")
```

Comment:
should we somewhere check the helm version? I guess the charts are versioned as well... ? Will that format somehow change in helm 3?

Reply:
We should reach out to someone on the Helm team to ask about how a Chart is flagged for working in v3 vs v2.  I assume v2 charts will work with v3, but not the other way around.  I'll reach out to someone on the Helm team and see if I can figure it out